packages/client/src/Views/PlayerUi/PlayerUi.tsx

```diff
@@ -78,6 +79,10 @@ const PlayerUi = ({ gameInfo, wargame, messageTypes, checkPasswordFail, loadData
 
 
   const handleCheckPassword = (pass: string): void => {
+    if (currentWargame.startsWith(hiddenPrefix)) {
```

Comment:
Hello @lebaphi - this test shouldn't fail, should it?

The hidden wargame shouldn't be listed, so it can't be present as `currentWargame`, can it.

I _guess_ this is handling some edge case.  Based on it being an edge case, I suggest the notification at line 21 below be changed to:
`Error: hidden wargame should not be available. Not opening.`

Reply:
Hi @IanMayo, the hidden wargame shouldn't be listed, right. This handles the case:
- in the client-side, the user loads the list of 3 games
- at this time on the server-side, the admin hide 1 of 3 games
- in the client-side, the user does not reload the page and chooses the hidden game then press enter.

=> we should handle this case and show an error message. I will update the error message.